lib/helpers/get-form-view-model.js

```diff
@@ -12,13 +11,20 @@ var cacheTimestamp = 0;
  *
  * @method
  *
+ * @param {string} formName - The name of the form (e.g. 'login' or 'register').
  * @param {number} ttl - The time-to-live value for the cached object.
  */
-function getCachedModel(ttl) {
-  var age = (new Date().getTime() - cacheTimestamp) / 1000; // In seconds.
+function getCachedModel(formName, ttl) {
+  var cacheItem = cache[formName];
 
-  if (cachedViewModel && age <= ttl) {
-    return cachedViewModel;
+  if (!cacheItem) {
+    return null;
```

Comment:
My habit, for situations where the return value is being used for a truthy test, is to use `return;`.  However I agree that `false` is more explicit and I like that.  If we're all in agreement I'd like to see an eslint rule that enforces no-null-returns, that should come as a separate PR after we clear this milestone.